geopandas/tests/test_geodataframe.py

```diff
@@ -806,6 +806,28 @@ def test_to_wkt(self):
         expected_df = pd.DataFrame({"gs0": wkts0, "gs1": wkts1})
         assert_frame_equal(expected_df, gdf.to_wkt())
 
+    @mock.patch("geopandas.io.arrow._to_parquet")
+    def test_to_parquet_fails_on_invalid_engine(self, mock_to_parquet):
```

Comment:
The `mock_to_parquet` is not used in this test?

Reply:
In addition, all other parquet-related tests are included in `geopandas/io/tests/test_arrow.py`. Can you move it there?

Reply:
Thanks for the update (and sorry for the slow follow-up!) 

I pushed a very small change: removing the mock usage in this test, because we can also use the tmpdir fixture to ensure any file that could incorrectly get written would be cleaned up. So then there is no worry about `_to_parquet` actually getting called. 
(for the other test, the use of `mock` is of course perfectly suited)